simulator/risc_v/t/unit_test.cpp

```diff
@@ -236,3 +236,21 @@ TEST_CASE( "RISV RV64 pack")
     CHECK( instr.get_v_dst() == 0x3333'3333'2222'2222);
 }
 
+TEST_CASE( "RISV RV32 xnor")
+{
+    CHECK( RISCVInstr<uint32>(0x40e6c633).get_disasm() == "xnor $a2, $a3, $a4");
+    RISCVInstr<uint32> instr ( "xnor", 0);
+    instr.set_v_src( 0x1234abcd, 0);
+    instr.set_v_src( 0xaaaaaaaa, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst() == 0x4761fe98);
```

Comment:
Could you please start with simpler examples?

Reply:
This example considers all possible cases of operations of this instruction.

Reply:
I see, that's great you have that goal. But simplicity is another target, for instance, it can be reached with another unit test. In the end, a person should be able to look at your unit tests and understand what that `xnor` does without looking up the manual. I believe it is quite easy for that instruction.